src/assets/scripts/client/App.js

```diff
@@ -114,9 +114,9 @@ export default class App {
         // This is provides a way to get async data from several sources in the app before anything else runs
         // FIXME: this is wrong. move this and make it less bad!
         $.when(
-            $.ajax(`assets/airports/${initialAirportToLoad.toLowerCase()}.json`),
-            $.ajax('assets/airlines/airlines.json'),
-            $.ajax('assets/aircraft/aircraft.json')
+            $.getJSON(`assets/airports/${initialAirportToLoad}.json`),
```

Comment:
could you add the back in `.toLowerCase()` here?  this is an extra defensiveness step because sometimes an icao is stored as `KSFO` vs `ksfo`.  this is a step at normalizing it before load.